# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/reftest.list
# Commit: d56eed692ac0
# Full Hash: d56eed692ac0a798b3d38e5ce1ebcdf928a1281e
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-30 09:39:43
# Description:
#   Bug 1771547 - Fix task graph dependencies with nested / sibling backdrop-filters r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147614
# ==============================================================================

diff -r 8e5ff7264e5d -r d56eed692ac0 gfx/wr/wrench/reftests/filters/reftest.list
--- a/gfx/wr/wrench/reftests/filters/reftest.list	Sun May 29 23:52:29 2022 +0000
+++ b/gfx/wr/wrench/reftests/filters/reftest.list	Mon May 30 06:17:30 2022 +0000
@@ -74,6 +74,7 @@
 == backdrop-filter-blur-edge-mode.yaml backdrop-filter-blur-edge-mode-ref.yaml
 == backdrop-filter-half-opacity.yaml backdrop-filter-half-opacity-ref.yaml
 == backdrop-filter-culled-filter.yaml backdrop-filter-culled-filter-ref.yaml
+== backdrop-filter-nested.yaml blank.yaml		# just don't crash
 platform(linux,mac) == backdrop-filter-transformed-filter.yaml backdrop-filter-transformed-filter.png
 platform(linux,max) == svg-filter-offset.yaml svg-filter-offset-ref.yaml
 skip_on(android,device) == fuzzy(1,100) svg-filter-composite.yaml svg-filter-composite-ref.yaml
